src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const daysPrice = days * price;
```

Comment:
I think it would be great to specify that the first variable is a rate for one day, and the second is the basic price without discounts